src/cmd-run

```diff
@@ -114,7 +114,16 @@ if [ -z "${IGNITION_CONFIG_FILE:-}" ]; then
             }
         ]
     },
-    "storage": {},
+    "storage": {
+        "files": [
+            {
+                "filesystem": "root",
```

Comment:
wish we could have comments in json files so we could explain why we did this without looking back at git history.